be/src/util/hdfs_util.cpp

```diff
@@ -64,6 +64,18 @@ std::string get_bucket_from_namenode(const std::string& namenode) {
     return namenode.substr(n, n2 - n);
 }
 
+std::string get_endpoint_from_oss_bucket(std::string& bucket, const std::string& default_bucket) {
```

Comment:
C++ coding style requires out parameter be pointer type, not reference type.

Reply:
done